http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/ProxyService.scala

```diff
@@ -284,7 +284,7 @@ class ProxyService(
     // 1. filter out headers not needed for the backend server
     val newHeaders = filterHeaders(request.headers)
     // 2. strip out Uri.Authority:
-    val newUri = Uri(path = rewrittenPath, queryString = request.uri.queryString())
+    val newUri = Uri(path = rewrittenPath, queryString = request.uri.rawQueryString)
```

Comment:
This is the actual fix: `queryString()` returns the _decoded_ query string, so we were mistakenly passing the decoded value to the Galaxy server